openfisca_nsw_safeguard/variables/ESS_PDRS_Estimator/SYS2/certificate_estimation/SYS2_PRC_calculation.py

```diff
@@ -58,14 +58,17 @@ def formula(buildings, period, parameters):
         network_loss_factor = buildings('SYS2_network_loss_factor', period)
         kw_to_0_1kw = 10
         replacement_activity = buildings('SYS2_replacement_activity', period)
+        input_power_eligibility = buildings('SYS2_input_power_PRCS_eligibility_int', period)
 
         SYS2_eligible_PRCs = np.select(
             [
-                replacement_activity,
-                np.logical_not(replacement_activity)
+                replacement_activity * input_power_eligibility,
+                np.logical_not(replacement_activity) * input_power_eligibility,
+                replacement_activity * np.logical_not(input_power_eligibility)
             ],
```

Comment:
I think we should also add a condition where they are both false 

so
np.logical_not(replacement_activity) * np.logical_not(input_power_eligibility)
and return 0



for both PRC and ESC calculation formula

Reply:
Will add it in now

Reply:
This is done.